GCRCatalogs/alphaq.py

```diff
@@ -60,6 +60,35 @@ def _calc_Av(lum_v, lum_v_dust):
     return Av
 
 
+def _gen_position_angle(size_reference):
```

Comment:
should cache `pos_angle` so that it doesn't regenerate every time the function is called. 

```python
def _gen_position_angle(size_reference):
    size = size_reference.size
    if not hasattr(_gen_position_angle, "_pos_angle") or _gen_position_angle._pos_angle.size != size:
        _gen_position_angle._pos_angle = np.random.RandomState(123497).uniform(0, 180, size)
    return _gen_position_angle._pos_angle
```

Reply:
Sorry about the messed up commit history.. I'm still getting a hang of git--I'll follow the instructions next time. :)